### DISCUSSION TITLE
Show Level Editor Box Color on blank frames in the level strip

### DISCUSSION THREAD

**Discussion by Nukiloco** [opentoonz/opentoonz#6818] [Ideas]
**Is your feature request related to a problem? Please describe**
When using the level strip and clicking on a new blank frame, the frame has a transparent background color and is not using Level Editor Box Color that I set as white. It will only set the background color as white after I'm done drawing a line on it.

**Describe the solution you'd like**
I would like it so that when I click on a new blank frame in the level strip, it will set the background color to the Level Editor Box Color without having to draw a line on it first.

**Comment by RodneyBaker** [comment]
@Flairieve Thanks for the report.
Yours is not the first time a request for a change related to this has appeared.
It would be good to collect as much user/workflow information as possible in order to move this forward to meet everyone's need.

A problem with changing the current approach is that some users appreciate the difference between level strip/editor and working on the canvas.  In short, seeing the difference apparently allows them to orient themselves quickly and know exactly where they are working (subconsciously) at an moment.

We'll get this smoothed out eventually but just have to get through the process...
